pointcloud/basic_octree_utils.go

```diff
@@ -38,23 +38,20 @@ func newLeafNodeFilled(p r3.Vector, d Data) basicOctreeNode {
 		children: nil,
 		nodeType: leafNodeFilled,
 		point:    PointAndData{P: p, D: d},
-		maxProb:  getRawProb(d),
+		maxVal:   getRawVal(d),
 	}
 	return octNode
 }
 
-// getRawProb returns the data param as a probability value.
-func getRawProb(d Data) float64 {
-	var maxProb float64
-	switch {
-	case !d.HasValue():
-		maxProb = 1
-	case d.Value() <= 100 && d.Value() >= 0:
-		maxProb = float64(d.Value()) / 100
-	default:
-		maxProb = emptyProb
+// getRawVal returns the data param as a probability value.
+func getRawVal(d Data) int {
+	var val int
+	if d.HasValue() {
+		val = d.Value()
+	} else {
+		val = emptyProb
 	}
-	return maxProb
+	return val
```

Comment:
very minor nit that this could be simplified to:

```
if d.hasValue() {
    return d.Value()
}
return emptyProb
```

Reply:
will change now.